wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -53,8 +53,9 @@ public function __construct( $id_base = '', $name = '', $widget_options = array(
 		$control_opts = wp_parse_args( $control_options, array() );
 
 		parent::__construct(
-			$id_base ? $id_base : 'wp-media-widget', // @todo This should just be 'media'.
-			$name ? $name : __( 'Media' ),
+			$id_base,
+			/* translators: placeholder is the id base of the widget */
+			$name ? $name : sprintf( __( 'Media: %s' ), $id_base ),
```

Comment:
So the media type would be lowercase and untranslatable? I think I'd prefer to either not make `$id_base` and `$name` optional instead